### DISCUSSION TITLE
How to change author avatar size in sidebar

### DISCUSSION THREAD

**Discussion by fritz-io** [mmistakes/minimal-mistakes#3721] [Q&A] [answered]
Issue: I want to enlarge the author avatar in the sidebar (I mean the circle should be larger).

My Github: https://github.com/fritz-io/fritz-io.github.io
My webpage: https://www.marvinfritz.de

I read that one should add this in main.scss but I have no idea on the correct syntax and the exact lines that should be changed. I read the issue #1709 and wanted to adapt it. I tried to add the following lines in main.scss, but nothing happened. I appreciate any input on this issue.

´@media {

.author__avatar {
  display: table-cell;
  vertical-align: top;
  width: 80px; //original: 36px
  height: 80px; //original: 36px

  @include breakpoint($large) {
    display: block;
    width: auto;
    height: auto;
  }

  img {
    max-width: 175px; //original: 110px
    border-radius: 50%;

    @include breakpoint($large) {
      padding: 5px;
      border: 1px solid $border-color;
    }
  }
}
}´

**Comment by fritz-io** [comment, marked as answer]
Sorry, I got it, I just had to add 
(min-width: 80em) 
after @media

**Reply by aayamrajshakya** [reply]
@fritz-io, how did you get the theme toggle working for you? Could you please help me?

**Comment by ldk-linux** [comment]
Did you figure out a way to increase the size on mobile/small screens? This works but only for large screens.
